### PR TITLE
Chrisk bugfixes

### PR DESCRIPTION
**Pull request by chriskr**

### CODE REVIEW

File: src/cookie-manager/cookie_manager_data.js [hunk 1]
```diff
@@ -41,15 +49,21 @@ cls.CookieManager.CookieDataBase = function()
...
   this.refetch = function()
   {
-    this._active_tab_count++;
     this.cookie_list = [];
-    for (var rt_id in this._rts) {
-      this._request_location_object_id(Number(rt_id), this._active_tab_count);
+    for (var rt_id in this._rts)
+    {
+      this._request_location_object_id(Number(rt_id));
     };
```
Comment (hzr) [inline src/cookie-manager/cookie_manager_data.js:56]: Remove the semicolon.

File: src/cookie-manager/cookie_manager_data.js [hunk 2]
```diff
@@ -131,29 +145,35 @@ cls.CookieManager.CookieDataBase = function()
...
-        this._rts[rt_id]={rt_id: rt_id};
-      }
-      this._request_location_object_id(rt_id, this._active_tab_count);
-    };
+        if (!this._rts[rt_id])
+        {
+          this._rts[rt_id] = {rt_id: rt_id};
+          this._request_location_object_id(rt_id);
+        }
+      };
```
Comment (hzr) [inline src/cookie-manager/cookie_manager_data.js:157]: Remove semicolon.

**Comment by danfooo** [comment]
Looks like the diff contains stuff that is already reviewed, the `bugfixes` branch has the `has_permanent` param for `addListener` in https://github.com/operasoftware/dragonfly/blob/bugfixes/src/scripts/messages.js for example.

**Comment by chriskr** [comment]
Merged with cutting-edge and run cleanrepo, now it should look better.

**Comment by danfooo** [comment]
Yes, much better.

**Comment by danfooo** [comment]
Looks fine. The fallback for displaying Authors is pretty cool I believe :)

**Comment by hzr** [comment]
Looks fine.

**Pull request merged**